src/afl_env.rs

```diff
@@ -63,35 +63,42 @@ impl AFLEnv {
         if let Some(ramdisk) = ramdisk {
             command.push(format!("AFL_TMPDIR={} ", ramdisk));
         }
+        if self.is_flag_enabled(&AFLFlag::AutoResume){
+            command.push(format!(
+                "AFL_AUTORESUME=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::FinalSync){
+            command.push(format!(
+                "AFL_FINAL_SYNC=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::DisableTrim){
+            command.push(format!(
+                "AFL_DISABLE_TRIM=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::KeepTimeouts){
+            command.push(format!(
+                "AFL_KEEP_TIMEOUTS=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::ExpandHavocNow){
+            command.push(format!(
+                "AFL_EXPAND_HAVOC_NOW=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::IgnoreSeedProblems){
+            command.push(format!(
+                "AFL_IGNORE_SEED_PROBLEMS=1 "
+            ));
+        }
+        if self.is_flag_enabled(&AFLFlag::ImportFirst){
+            command.push(format!(
+                "AFL_IMPORT_FIRST=1 "
+            ));
+        }
```

Comment:
Since you are more proficient with rust can you maybe improve this by removing the duplicate code and just have a flag -> env-variable mapping (string) ?

Reply:
Something like this is likely easier to extend/maintain:

```rust
impl ToString for AFLFlag {
    fn to_string(&self) -> String {
        match self {
            AFLFlag::AutoResume => "AFL_AUTORESUME",
            AFLFlag::FinalSync => "AFL_FINAL_SYNC",
            AFLFlag::DisableTrim => "AFL_DISABLE_TRIM",
            AFLFlag::KeepTimeouts => "AFL_KEEP_TIMEOUTS",
            AFLFlag::ExpandHavocNow => "AFL_EXPAND_HAVOC_NOW",
            AFLFlag::IgnoreSeedProblems => "AFL_IGNORE_SEED_PROBLEMS",
            AFLFlag::ImportFirst => "AFL_IMPORT_FIRST",
        }
        .to_string()
    }
}

impl FromStr for AFLFlag {
    type Err = String;

    fn from_str(s: &str) -> Result<Self, Self::Err> {
        match s {
            "AFL_AUTORESUME" => Ok(AFLFlag::AutoResume),
            "AFL_FINAL_SYNC" => Ok(AFLFlag::FinalSync),
            "AFL_DISABLE_TRIM" => Ok(AFLFlag::DisableTrim),
            "AFL_KEEP_TIMEOUTS" => Ok(AFLFlag::KeepTimeouts),
            "AFL_EXPAND_HAVOC_NOW" => Ok(AFLFlag::ExpandHavocNow),
            "AFL_IGNORE_SEED_PROBLEMS" => Ok(AFLFlag::IgnoreSeedProblems),
            "AFL_IMPORT_FIRST" => Ok(AFLFlag::ImportFirst),
            _ => Err(format!("Unknown AFL flag: {}", s)),
        }
    }
}

//[...]

impl AFLEnv {
   //[...]


       /// Generates an `AFLPlusPlus` environment variable string for the current settings
    pub fn generate_afl_env_cmd(&self, ramdisk: Option<String>) -> Vec<String> {

        [...]
        for flag in &self.flags {
            command.push(format!("{}={}", flag.to_string(), 1));
        }

        command.push(format!("AFL_TESTCACHE_SIZE={}", self.testcache_size));
```

Reply:
Thanks a lot! Pushed the changes you suggested